charts/argo-cd/values.yaml

```
@@ -500,6 +500,7 @@ server:
     domain: argocd.example.com
     issuer: {}
```

While reviewing this I thought about adding this (It got removed in a very old PR https://github.com/argoproj/argo-helm/pull/129):
```suggestion
    issuer:
      kind: # ClusterIssuer
      name: # letsencrypt
```
WDYT?

---

It can't hurt, I tried templating it and it doesn't break anything either so I have added it.